src/state/selection/reducer.ts

```diff
@@ -33,6 +35,7 @@ export const initialState = {
     agentHighlightMap: {},
     colorChange: null,
     recentColors: [],
+    selectedAgent: {},
```

Comment:
nit, but I would call this `selectedAgentMetadata` and the selector then `getSelectedAgentMetadata`

Reply:
Yeah that's definitely better.